File: Online_Judge_Problems/BOJ/boj_11943/boj_11943.js
/*
----------------------------------------------
Sub : [BOJ] 파일 옮기기
Link: https://www.acmicpc.net/problem/11943
Tag : JS, 
Memo
----------------------------------------------
*/

const fs = require('fs')
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n')

const [a, b] = input[0].split(" ").map(Number)
const [c, d] = input[1].split(" ").map(Number)

const result = b+c < a+d ? b+c : a+d

console.log(result)

File: Online_Judge_Problems/BOJs/boj_11943/boj_11943.py
'''
----------------------------------------------
Sub : [BOJ] 파일 옮기기
Link: https://www.acmicpc.net/problem/11943
Tag : Python, 
Memo
----------------------------------------------
'''

a, b = map(int, input().split())
c, d = map(int, input().split())

if (b+c < a+d):
    result = b+c
else :
    result = a+d

print(result)
